### ISSUE TITLE
Feature: Display /me messages (XEP-0245) differently from ordinary ones

### ISSUE THREAD

**Issue by badrihippo** [siacs/Conversations#2800]
***TL;DR:** Remove the white/green background from `/me` messages, or use some other strategy to differentiate them from ordinary messages.*

The current behaviour of the `/me` command is to simply prepend the sender's name in bold-italics. It would be nice if they were given a slightly different styling as well, to make them stand out.

Clients generally display `/me` messages something like

>  \* *badrihippo thinks...*

...while ordinary messages look like

> **badrihippo:** I think...

This enhances the effect of the `/me` command, and makes it more fun to use than simply typing ones' name.

The Conversations UI is different from most clients, so we'll need to use a different strategy to highlight the `/me` commands. A start would be to remove the white/green speech-bubble background, making the message merge with the background like this (before and after shown below):

![conversation-resized](https://user-images.githubusercontent.com/7288084/36257347-7a8f39c6-127c-11e8-8c29-fb4464378f95.png)

The text could also be made italic and/or muted, but it shouldn't look too much like a system message (eg. "so-and-so has stopped typing")

**Comment by iNPUTmice** [comment]
Not a huge fan to be honest. I think it's irritating that the bubble is missing. Anyway `/me` is not a very common feature these days any way. So I don’t think we need to focus on improving this.
